tools/reccmp/reccmp.py

```diff
@@ -383,14 +471,21 @@ def parse_asm(file, addr, size):
 
           function_count += 1
           total_accuracy += ratio
+          total_effective_accuracy += effective_ratio
 
           if recinfo.size:
             udiff = difflib.unified_diff(origasm, recompasm, n=10)
 
             # If verbose, print the diff for that funciton to the output
             if verbose:
-              if ratio == 1.0:
-                print("%s: %s 100%% match.\n\nOK!" % (hex(addr), recinfo.name))
+              if effective_ratio == 1.0:
+                ok_text = "OK!" if plain else (colorama.Fore.GREEN + "✨ OK! ✨" + colorama.Style.RESET_ALL)
```

Comment:
You really want these emoji don't you lol, I saw them in #74 too

Reply:
More dopamine = more contributions. Can't argue with that.